pkg/console/subresource/deployment/deployment.go

```diff
@@ -221,11 +221,11 @@ func GetLogLevelFlag(logLevel operatorv1.LogLevel) string {
 	flag := ""
 	switch logLevel {
 	case operatorv1.Normal:
-		flag = "--log-level=*=NOTICE"
+		flag = "ERROR"
 	case operatorv1.Debug:
-		flag = "--log-level=*=DEBUG"
+		flag = "INFO"
 	case operatorv1.Trace, operatorv1.TraceAll:
-		flag = "--log-level=*=TRACE"
```

Comment:
Based on the [klog docs](https://github.com/kubernetes/klog/blob/master/klog.go#L98-L100) I would suggest to set the severity as:

```go
	case operatorv1.Normal:
		flag = "INFO"
	case operatorv1.Debug:
		flag = "WARNING"
	case operatorv1.Trace:
                flag = "ERROR"
        case operatorv1.TraceAll:
		flag = "FATAL"
	}
```

Reply:
Those seem reasonable; the first round were mainly placeholders to discuss how to best map them in the future.  Thanks for the input.

Reply:
Does this work? Looking at [line 250](https://github.com/openshift/console-operator/pull/250/files#diff-8c5d48b60d928752e02d8916d831a48eR250), typically flags are as:

```
--key=value
```
So we are turning 
```
--log-level=*=NOTICE
```
into just

```
INFO
```
Seems like this will create an invalid command in our deployment func:

```
		Command: []string{
			"/opt/bridge/bin/bridge",
			"--public-dir=/opt/bridge/static",
			"--config=/var/console-config/console-config.yaml",
			"--service-ca-file=/var/service-ca/service-ca.crt",
			flag,
		},
```
Make sure you test again with a log level explicitly set, not sure this is quite right.

Reply:
IE, if `oc apply -f` a file like this:

```
apiVersion: operator.openshift.io/v1
kind: Console
metadata:
  name: cluster
spec:
  managementState: Managed
  logLevel: Trace
```
Does the console pod still run?

Reply:
I believe the new log level should be something like 

```
 -v=X
``` 
Similar to the `console-operator` config itself.  You can find this in [manifests/07-operator.yaml](https://github.com/openshift/console-operator/blob/master/manifests/07-operator.yaml)

```
command:
- console
- operator
args:
# this is the operator's klog setting
- "-v=6"
- "--config=/var/run/configmaps/config/controller-config.yaml"
```

Reply:
So, given the above, I think the `GetLogLevel()` func probably needs to be more like this:

```golang
func GetLogLevelFlag(logLevel operatorv1.LogLevel) string {
	flag := ""
	switch logLevel {
	case operatorv1.Normal:
		flag = "-v=2"
	case operatorv1.Debug:
		flag = "-v=4"
	case operatorv1.Trace:
		flag = "-v=6"
	case operatorv1.TraceAll:
		flag = "-v=10"
	}
	return flag
}
```